javascript.ru/2.8.1/2.8.1.js

```diff
@@ -0,0 +1,16 @@
+﻿var a = 1, b = 1, c, d;
+
+c = ++a;
+alert(c);
+
+d = b++;
+alert(d);
+
+c = (2 + ++a);
+alert(c);
+
+d = (2 + b++);
+alert(d);
+
+alert(a); 
+alert(b);
```

Comment:
Don't do output in same way. Try always to write something like "a = 1" and "b = 2" in all your alerts.

Reply:
Done commit ecda81538788753df057116e516d036ebf128399